client/components/Dashboard/RecentCohorts.jsx

```diff
@@ -0,0 +1,70 @@
+import './Dashboard.css';
+
+import { Button, Icon } from '@components/UI';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import CohortCard from '../Cohorts/CohortCard';
+import { SCENARIO_IS_PUBLIC } from '@components/Scenario/constants';
+import { getRecentCohorts } from '@actions/cohort';
+import { getScenariosByStatus } from '@actions/scenario';
+
+const CohortContent = ({ cohorts }) => {
+  if (cohorts.length) {
+    return (
+      <ul className="dashboard-grid">
+        {cohorts.map(cohort => {
+          return (
+            <li key={cohort.id}>
+              <CohortCard id={cohort.id} />
+            </li>
+          );
+        })}
+      </ul>
+    );
+  } else {
+    return (
+      <div className="dashboard-empty">
+        <p>
+          No cohorts created. <a href="#">Create a new cohort.</a>
+        </p>
+      </div>
+    );
```

Comment:
I suspect this whole else statement block is going to change, according to @enatario's feedback here: https://github.com/mit-teaching-systems-lab/dcss/pull/11#issuecomment-1123748896, but when implementing that, please be sure to use the appropriate semantic ui component. Thanks!